### ISSUE TITLE
IPC broke again ? 

### ISSUE THREAD

**Issue by ang-st** [ava-labs/avalanchego#1063]
**Describe the bug**
avalanchego does not send any data on the ipcs unix sockets 

**To Reproduce**
run 
```
./avalanchego  --http-host 0.0.0.0 --api-ipcs-enabled=true --network-id=1  --api-admin-enabled=false  --api-keystore-enabled=false --ipcs-chain-ids=2oYMBNV4eNHyqk2fjjV5nVQLDbtmNJzq5s3qs3Lo6ftnC6FByM
```
connect a client to it : 
```
nc -U /tmp/1-2oYMBNV4eNHyqk2fjjV5nVQLDbtmNJzq5s3qs3Lo6ftnC6FByM-consensus
````
no data are spitted out

**Expected behavior**

getting ipcs message  defined in https://docs.avax.network/build/avalanchego-apis/ipc-api#ipc-message-format

**Operating System**

```
cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.5 LTS"
```

```
uname -a
Linux AVAX-sweet-banzai 4.15.0-154-generic #161-Ubuntu SMP Fri Jul 30 13:04:17 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

I noticed while upgrading to `avalanchego-v1.7.10`

interestingly if i let socket open long enough we get some tx and some other stuff passing through ...
![image](https://user-images.githubusercontent.com/1777820/167905729-5b430b05-66c1-4eb5-af5f-30a2eafe4620.png)

**Comment by StephenButtolph** [comment]
This will be resolved by v1.7.11. Sorry for the regression. v1.7.11 will also introduce some additional structure to avoid any errors like this in the future.

**Comment by ang-st** [comment]
@StephenButtolph when is v1.7.11 scheduled to release ?

**Comment by StephenButtolph** [comment]
Didn't mean to close this - was closed automatically via an internal merge. v1.7.11 is currently being tested on Fuji: https://explorer-xp.avax-test.network/validators. That's really as much of a timeline as I can hope to give out...

**Comment by Activatorq1** [comment]
Kindly open this live chat link to talk to the customer service directly on your issue  https://direct.lc.chat/14156445/

**Comment by StephenButtolph** [comment]
Closing as this should have been resolved with the release of v1.7.11
